core/src/main/java/org/teavm/model/MethodHandleType.java

```diff
@@ -16,6 +16,7 @@
 package org.teavm.model;
 
 public enum MethodHandleType {
+    REF_NONE,
```

Comment:
For what purpose? How are you going to use it? Found no references to this field. Also, adding such things in the core model requires **a lot** of things to be changed, like LambdaMetatafactorySubstitutor, model serializer and deserializer, etc, etc.

Reply:
This was to align with the standard Java library. I've added a referenceKind value instead now on the enum so it will be backwards compatible.